package.json

```diff
@@ -3,53 +3,40 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "config:dev": "ts-node --project ./tsconfig.ts-node.json scripts/set-env.ts --dev",
-    "config:prod": "ts-node --project ./tsconfig.ts-node.json scripts/set-env.ts --prod",
-    "config:test": "ts-node --project ./tsconfig.ts-node.json scripts/set-mock-env.ts",
-    "config:test:watch": "nodemon --config mock-nodemon.json",
-    "config:dev:watch": "nodemon",
-    "config:check:rest": "yarn run config:prod && ts-node --project ./tsconfig.ts-node.json scripts/test-rest.ts",
-    "config:dev:check:rest": "yarn run config:dev && ts-node --project ./tsconfig.ts-node.json scripts/test-rest.ts",
-    "prestart:dev": "yarn run config:dev",
-    "prebuild": "yarn run config:dev",
-    "pretest": "yarn run config:test",
-    "pretest:watch": "yarn run config:test",
-    "pretest:headless": "yarn run config:test",
-    "prebuild:prod": "yarn run config:prod",
-    "pree2e": "yarn run config:prod",
+    "config:watch": "nodemon",
+    "test:rest": "ts-node --project ./tsconfig.ts-node.json scripts/test-rest.ts",
     "start": "yarn run start:prod",
-    "serve": "ts-node --project ./tsconfig.ts-node.json scripts/serve.ts",
-    "start:dev": "npm-run-all --parallel config:dev:watch serve",
-    "start:prod": "yarn run build:prod && yarn run serve:ssr",
+    "start:dev": "nodemon --exec \"yarn run serve\"",
```

Comment:
Shouldn't this contain `NODE_ENV=development` similar to  `start:prod`? currently my production config is used when I run `start:dev`

Reply:
Yeah, thanks. Good catch.

Reply:
Added.